testing/test_pytest_html.py

```diff
@@ -442,7 +444,7 @@ def pytest_runtest_makereport(item, call):
         """.format(extra_type, content))
         testdir.makepyfile("""
             import pytest
-            @pytest.mark.flaky(reruns=2)
+            @pytest.mark.flaky(reruns=4)
```

Comment:
Just because the loop bellow goes from 1 to 4 (so it checks for extra content 4 times) and it was only 2 reruns.

Reply:
I see your logic, but I wonder why we had this mismatch with tests passing? This code hasn't changed in four years.

Reply:
I saw the logic there before, actually the loop checks for content 3 times (`range(1,4) = [1, 2, 3]`, my fault) and the first test (the one which failed) count as the file1, the first rerun is actually  file2, and so on